src/site/site.xml

```diff
@@ -39,6 +39,8 @@
 
   <body>
     <breadcrumbs>
+      <item name="Plexus" href="https://codehaus-plexus.github.io/"/>
+      <item name="Parent POM" href="https://codehaus-plexus.github.io/plexus-pom"/>
```

Comment:
Do we really need a link to parent in breadcrumbs? 
We have a link to parent in menu items ...
@hboutemy @michael-o